sandbox/cri/config.toml

```diff
@@ -0,0 +1,189 @@
+# Firecracker
+[proxy_plugins]
+  [proxy_plugins.firecracker-naive]
+    type = "snapshot"
+    address = "/var/run/firecracker-containerd/naive-snapshotter.sock"
+  [proxy_plugins.firecracker-dm-snapshotter]
+    type = "snapshot"
+    address = "/var/run/firecracker-dm-snapshotter.sock"
```

Comment:
Any reason the paths are inconsistent for the dm snapshotter and naive snapshotter?